### PR TITLE
Allow Command to parse Bors and bors

### PR DESCRIPTION
**Pull request by GustavoCaso**
#308 

I decided to use `Regex` in this case because it seems the most simple way to keep functionality without changing too much code.

@notriddle please add as many comments as need it.

### CODE REVIEW

File: apps/bors_frontend/web/command.ex [hunk 1]
```diff
@@ -107,20 +107,19 @@ defmodule BorsNG.Command do
...
-      @command_trigger <> ": " <> cmd ->
-        trim_and_parse_cmd(cmd)
-      _ -> []
+    |> Enum.flat_map(fn(string) ->
+      trim_and_parse_cmd(Regex.named_captures(regex(), string))
     end)
   end
 
-  def trim_and_parse_cmd(cmd) do
+  def regex, do: ~r/^#{@command_trigger}:?\s(?<command>.+)/i
```
Comment (Macarse) [inline apps/bors_frontend/web/command.ex]: elixir doesn't offer a way to compile regex, right?

Comment (notriddle) [reply apps/bors_frontend/web/command.ex]: I think [this *is* the way to compile regex](https://hexdocs.pm/elixir/Regex.html).

> Regular expressions created via sigils are pre-compiled and stored in the `.beam` file.

**Comment by notriddle** [comment]
Just as a quick tip, @GustavoCaso, if you include `closes #308`, instead of just `#308`, in your issue number, then the issue will automatically be closed as soon as your PR gets merged.

**Comment by GustavoCaso** [comment]
@notriddle awesome thanks for the quick tip

**Pull request merged**